Supervision Work/src/uk/ac/cam/ap886/oopjava/supervisionProjects/Hangman/Game.java

```diff
@@ -0,0 +1,115 @@
+package uk.ac.cam.ap886.oopjava.supervisionProjects.Hangman;
+
+import java.util.ArrayList;
+
+public class Game {
+	protected 	Dictionary 	dictionary;		
+	protected 	Player 		player;		  	
+	protected 	char[] 		etalon;			
```

Comment:
This seems like a fairly obscure variable name, which I feel could be named more helpfully or at least have a comment next to the declaration explaining what it is